htsohm/htsohm.py

```diff
@@ -36,53 +37,64 @@ def queue_all_materials(run_id, generation, queue):
     for material in materials:
         queue.enqueue(hpc_job_run_all_simulations, material.id, timeout=60*60)
 
-def seed_generation(run_id, children_per_generation, number_of_atomtypes, queue=None):
+def create_next_generation(run_id, generation):
+    children_per_generation = read_config_file(run_id)['children-per-generation']
+    for i in range(children_per_generation):
+        parent_id = screen_parent(run_id)
+        write_child_definition_files(run_id, generation, parent_id)
+    session.commit()
+
+def hpc_job_create_material(run_id, generation):
+    print("======================================================================================")
+    print("== manhpc_job_dummy_test %s" % material_id)
+
+    parent_id = screen_parent(run_id)
+    write_child_definition_files(run_id, generation, parent_id)
+    session.commit()
+    print("======================================================================================")
+
+def queue_create_next_gen(run_id, generation, queue):
+    """same as create_all_materials, except queues the jobs in the job server"""
```

Comment:
If you're going to call the normal method create_next_generation, just call the queue method queue_create_next_generation. 

Also, there is no "create_all_materials" method